sql/core/src/main/scala/org/apache/spark/sql/SQLContext.scala

```diff
@@ -328,18 +328,19 @@ class SQLContext(@transient val sparkContext: SparkContext)
     def numPartitions = self.numShufflePartitions
 
     def strategies: Seq[Strategy] =
-      extraStrategies ++ (
-      CommandStrategy ::
-      DataSourceStrategy ::
-      TakeOrdered ::
-      HashAggregation ::
-      LeftSemiJoin ::
-      HashJoin ::
-      InMemoryScans ::
-      ParquetOperations ::
-      BasicOperators ::
-      CartesianProduct ::
-      BroadcastNestedLoopJoin :: Nil)
+      extraStrategies ++ Seq(
+      CommandStrategy,
+      DDLStrategy(self),
```

Comment:
Sorry, what does `move DDLStrategy ahead of CommandStrategy` means: change the order to ?
```
DDLStrategy(self),
CommandStrategy,
```
scfw's idea is to `remove` this `DDLStrategy` and use `execution.DDLDescribeCommand` instead but not `move`.